dashboard/lib/services/child_account/event_logger.rb

```diff
@@ -8,6 +8,7 @@ class EventLogger < Services::Base
       # def self.log_account_locking(user)
       # def self.log_account_purging(user)
       # def self.log_compliance_removing(user)
+      # def self.log_sections_warning_email_sent(user)
```

Comment:
Please add a unit test for the method similar to this: https://github.com/code-dot-org/code-dot-org/blob/c53cba7586d76c248d638c7598e4ade848a32519/dashboard/test/lib/services/child_account/event_logger_test.rb#L73-L77